### PR TITLE
add friendly id and some bug fixes

### PR DESCRIPTION
**Pull request by ninalouw**
@thaisamarinho 
@mrreubenjones 
Will you guys please take a look?

### CODE REVIEW

File: app/models/sighting.rb [hunk 1]
```diff
@@ -4,4 +4,13 @@ class Sighting < ApplicationRecord
   validates :last_seen_at, presence: true
   validates :date_time, presence: true
   mount_uploader :image, ImageUploader
+
+  extend FriendlyId
+  friendly_id :slug_candidates, use: [:slugged, :history]
+  def slug_candidates
+    [
+      :pet_type,
```
Comment (reuben-jones) [inline app/models/sighting.rb]: Would consider removing this line, as the pet_type would be populated pretty quick.

**Review by reuben-jones — APPROVED** [review]
Looks awesome. 

See note in the sighting.rb file, but it's just a minor thing.

**Comment by reuben-jones** [comment]
@jeffcstock @ninalouw 

We think it's just wonderful. Feel free to send on to Steve. :)

**Comment by jeffcstock** [comment]
Thanks for reviewing @mrreubenjones @thaisamarinho, now it's your turn @whilestevego!

File: app/models/sighting.rb [hunk 2]
```diff
@@ -4,4 +4,10 @@ class Sighting < ApplicationRecord
   validates :last_seen_at, presence: true
   validates :date_time, presence: true
   mount_uploader :image, ImageUploader
+
+  extend FriendlyId
```
Comment (whilestevego) [inline app/models/sighting.rb]: Can you move this line to the top of the class?

File: app/models/sighting.rb [hunk 3]
```diff
@@ -4,4 +4,10 @@ class Sighting < ApplicationRecord
...
+  friendly_id :slug_candidates, use: [:slugged, :history]
```
Comment (whilestevego) [inline app/models/sighting.rb:10]: Add an empty line after this one to separate it from the method below

File: config/initializers/friendly_id.rb [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+  # FriendlyId uses Rails's `parameterize` method to generate slugs, but for
+  # languages that don't use the Roman alphabet, that's not usually sufficient.
+  # Here we use the Babosa library to transliterate Russian Cyrillic slugs to
+  # ASCII. If you use this, don't forget to add "babosa" to your Gemfile.
+  #
+  # config.use Module.new {
+  #   def normalize_friendly_id(text)
+  #     text.to_slug.normalize! :transliterations => [:russian, :latin]
+  #   end
+  # }
```
Comment (whilestevego) [inline config/initializers/friendly_id.rb]: Can you delete on the info comments in here?

**Review by whilestevego — CHANGES_REQUESTED** [review]
Looking good. There are only a few changes to do

**Comment by jeffcstock** [comment]
Please review our edits again @whilestevego.

**Review by whilestevego — APPROVED** [review]

**Pull request merged**